selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/workspaces/WorkspaceOverview.java

```diff
@@ -87,6 +109,119 @@ public void checkNameWorkspace(String nameWorkspace) {
+        By.xpath(SAVE_BUTTON_XPATH), "aria-disabled", "false");
+  }
+
+  public void waitExportWsFormOpened() {
+    seleniumWebDriverHelper.waitVisibility(By.xpath(EXPORT_WS_FORM_XPATH));
+  }
+
+  public String getAsFileFormText() {
+    return seleniumWebDriverHelper.waitVisibilityAndGetText(By.xpath(AS_FILE_CONFIG_BODY_XPATH));
+  }
+
+  public void waitConfiguration(String... expectedConfiguration) {
+    asList(expectedConfiguration).forEach(config -> getAsFileFormText().contains(config));
+  }
+
+  public void clickOnAsFileButton() {
+    seleniumWebDriverHelper.waitAndClick(By.xpath(AS_FILE_BUTTON_XPATH));
+  }
+
+  public void waitExportWorkspaceFormOpened() {
+    seleniumWebDriverHelper.waitVisibility(By.xpath(EXPORT_WS_FORM_XPATH));
+    testWebElementRenderChecker.waitElementIsRendered(By.xpath(EXPORT_WS_FORM_XPATH));
+  }
+
+  public void waitExportWorkspaceFormClosed() {
+    seleniumWebDriverHelper.waitInvisibility(By.xpath(EXPORT_WS_FORM_XPATH));
+  }
+
+  public boolean isAsFileTabOpened() {
+    return seleniumWebDriverHelper
+        .waitVisibilityAndGetAttribute(
+            By.xpath(AS_FILE_BUTTON_XPATH + "/parent::md-tab-item"), "aria-selected")
+        .equals("true");
+  }
+
+  public void waitAsFileTabOpened() {
+    seleniumWebDriverHelper.waitSuccessCondition(driver -> isAsFileTabOpened());
+  }
+
+  public void waitAsFileTabClosed() {
+    seleniumWebDriverHelper.waitSuccessCondition(driver -> !isAsFileTabOpened());
+  }
+
+  public void clickOnToPrivateCloudButton() {
+    seleniumWebDriverHelper.waitAndClick(By.xpath(To_PRIVATE_CLOUD_BUTTON_XPATH));
+  }
+
+  public boolean isToPrivateCloudTabOpened() {
+    return seleniumWebDriverHelper
+        .waitVisibilityAndGetAttribute(
+            By.xpath(To_PRIVATE_CLOUD_BUTTON_XPATH + "/parent::md-tab-item"), "aria-selected")
+        .equals("true");
+  }
+
+  public void waitToPrivateCloudTabOpened() {
+    seleniumWebDriverHelper.waitSuccessCondition(driver -> isToPrivateCloudTabOpened());
+  }
+
+  public void waitToPrivateCloudTabClosed() {
+    seleniumWebDriverHelper.waitSuccessCondition(driver -> !isToPrivateCloudTabOpened());
+  }
+
+  public void clickOnDownloadButton() {
+    final String downloadButtonXpath = "//che-button-default [@che-button-title='download']";
+    seleniumWebDriverHelper.waitAndClick(By.xpath(downloadButtonXpath));
+  }
+
+  public void clickOnClipboardButton() {
+    final String clipboardButtonXpath = "//che-button-default [@che-button-title='clipboard']";
+    seleniumWebDriverHelper.waitAndClick(By.xpath(clipboardButtonXpath));
+  }
+
+  public void clickOnCloseExportWorkspaceButton() {
+    final String closeButtonXpath = "//che-button-notice [@che-button-title='Close']";
+    seleniumWebDriverHelper.waitAndClick(By.xpath(closeButtonXpath));
+  }
+
+  public void clickOnCloseExportWorkspaceIcon() {
+    seleniumWebDriverHelper.waitAndClick(By.xpath(EXPORT_WS_FORM_XPATH + "//i"));
```

Comment:
It's not quite clear which element xpath `TEXPORT_WS_FORM_XPATH + "//i"` is linked to.